src/components/Recipes/Recipes.tsx

```diff
@@ -18,7 +20,7 @@ const Recipes: React.FC = () => {
   useEffect(() => {
     if(ingredientList.length !== 0) {
 
-      debounce(fetchData, 1400, itemsReset(), addRecipe, ingredientList, timeout)
+      debounce(fetchData, 1400, itemsReset(), addRecipe, setRecipeSearchResults, ingredientList, timeout)
```

Comment:
is this breaking the debounce -- adding an extra argument? -- i dont see changes to the debounce function

@cguizar25

Reply:
This look like it was to replace the temporary static amount of items that the debouncer sets, the variable that's actually using that argument is the function that's being used as an argument here, so you'll see the relevant change in SpoonacularRequests.

Reply:
Right. The debounce is wrapping the request function and this was to pass an extra argument to the request.